### PR TITLE
add volume mount docs for ticdc

### LINKED ISSUE #1717: Add description about how to configure volumes for TiCDC
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.
Add a description about how to configure volumes for TiCDC, it's not covered in https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/deploy-ticdc.
2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by just1900**
Signed-off-by: just1900 <legendj228@gmail.com>

### What is changed, added, or deleted? (Required)

close #1717 
add volume mount docs for ticdc

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-ticdc.md [hunk 1]
```diff
@@ -92,3 +92,38 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-ticdc/']
     ```
 
 如果服务器没有外网，请参考 [部署 TiDB 集群](deploy-on-general-kubernetes.md#部署-tidb-集群) 在有外网的机器上将用到的 Docker 镜像下载下来并上传到服务器上。
+
+## 为 TiCDC 挂载 PV
```
Comment (ran-huang) [inline zh/deploy-ticdc.md:96]: 1. 挂载 PV 是不是应该在创建集群之前？
2. 这一节看起来跟[多盘挂载](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/configure-a-tidb-cluster#%E5%A4%9A%E7%9B%98%E6%8C%82%E8%BD%BD)大致一样，能否直接链接过去，并在 configure-a-tidb-cluster.md 里加入 ticdc 的示例（用 simpletab）

Comment (just1900) [reply zh/deploy-ticdc.md:96]: lgtm, will do.

File: zh/configure-a-tidb-cluster.md [hunk 2]
```diff
@@ -117,6 +121,69 @@ TiDB Operator 支持为 PD、TiDB、TiKV、TiCDC 挂载多块 PV，可以用于
...
+    ...
+    config:
+      dataDir: /ticdc/data
+      logFile: /ticdc/log/cdc.log
+    storageVolumes:
+    - name: data
+      storageSize: "10Gi"
+      storageClassName: local-storage
+      mountPath: "/ticdc/data"
+    - name: log
```
Comment (csuzhangxc) [inline zh/configure-a-tidb-cluster.md:177]: 上面 TiDB/PD 的例子，名字也起个更有意义点的吗 😂

**Review by csuzhangxc — APPROVED** [review]

File: zh/deploy-ticdc.md [hunk 3]
```diff
@@ -45,15 +45,17 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-ticdc/']
        baseImage: pingcap/ticdc-enterprise
    ```
 
-3. 部署完成后，通过 `kubectl exec` 进入任意一个 TiCDC Pod 进行操作。
+3. 为 TiCDC 挂载 PV：TiCDC 支持挂载多个 PV，建议第一次部署 TiCDC 时规划好使用几个 PV，相关配置请参阅[多盘挂载](configure-a-tidb-cluster.md#多盘挂载)
```
Comment (ran-huang) [inline zh/deploy-ticdc.md:48]:
```suggestion
3. 为 TiCDC 挂载 PV。

    TiCDC 支持挂载多个 PV，建议在第一次部署 TiCDC 时规划好使用几个 PV。相关配置请参阅[多盘挂载](configure-a-tidb-cluster.md#多盘挂载)。
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Review by csuzhangxc — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**